<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>260</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>汉仪文黑-85W</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪文黑-85W</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>用户名</string>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪文黑-85W</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>密码</string>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="LoginButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪文黑-85W</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toRegister">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪文黑-85W</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="remember">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>30</y>
     <width>190</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>username</tabstop>
  <tabstop>password</tabstop>
  <tabstop>remember</tabstop>
  <tabstop>LoginButton</tabstop>
  <tabstop>toRegister</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
